qualimap/qualimap.cwl

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.0
+class: CommandLineTool
+
+# Metadata
+label: qualimap-qc
+doc: This is qualimap CWL tool definition http://qualimap.bioinfo.cipf.es/
+
+hints:
+  ResourceRequirement:
+    ramMin: 4000
+    coresMin: 1
+  SoftwareRequirement:
+    packages:
+      qualimap:
+        version: [ "2.2.2d" ]
+        specs:
+          - https://identifiers.org/rrid/RRID:SCR_001209
+          - https://bio.tools/qualimap
+  DockerRequirement:
+    dockerPull: 'quay.io/biocontainers/qualimap:2.2.2d--1'
+requirements:
+  - class: ShellCommandRequirement
+
+# Base command
+baseCommand: [ "qualimap"]
```

Comment:
```suggestion
baseCommand: [ qualimap, rnaseq ]
```
I would also request the file be renamed `qualimap_rnaseq.cwl` to allow for the other sub commands as well

Reply:
@mr-c - done in [d7e7611](https://github.com/common-workflow-library/bio-cwl-tools/pull/78/commits/d7e76111a47caae606dca712afa2cf6b015da327)